kuma/wiki/tests/test_models.py

```diff
@@ -601,19 +601,24 @@ def test_correct_based_on_to_current_revision(self):
     def test_previous(self):
         """Revision.previous should return this revision's document's
         most recent approved revision."""
-        rev = revision(is_approved=True, save=True)
-        eq_(None, rev.previous)
-        # wait a second so next revision is a different datetime
-        time.sleep(1)
+        rev = revision(is_approved=True, created=datetime(2017, 4, 15, 9, 23),
```

Comment:
If you rewrite the [model method](https://github.com/mozilla/kuma/blob/master/kuma/wiki/models.py#L1804) queryset like this
```python
try:
    return self.document.revisions.filter(is_approved=True, id__lt=self.id).latest()
except self. DoesNotExist:
    return None```

Its possible to remove the ``time.sleep()`` easily